dashboard/app/email_test.go

```diff
@@ -41,7 +42,7 @@ func TestEmailReport(t *testing.T) {
 		kernelConfigLink := externalLink(c.ctx, textKernelConfig, dbBuild.KernelConfig)
 		c.expectEQ(sender, fromAddr(c.ctx))
 		to := config.Namespaces["test2"].Reporting[0].Config.(*EmailConfig).Email
-		c.expectEQ(msg.To, []string{to})
+		c.expectEQ(msg.To, []string{"crashCC@foo.com", to})
```

Comment:
This looks wrong. We don't send email to crash.Maintainers here, but we added crash.Cc addresses.